airflow/migrations/versions/cf5dc11e79ad_drop_user_and_chart.py

```diff
@@ -47,7 +47,12 @@ def upgrade():  # noqa: D103
 
     if 'known_event' in tables:
         for fkey in inspector.get_foreign_keys(table_name="known_event", referred_table="users"):
-            op.drop_constraint(fkey['name'], 'known_event', type_="foreignkey")
+            if conn.dialect.name == "sqlite":
+                if fkey['name']:
+                    with op.batch_alter_table(table_name='known_event') as bop:
+                        bop.drop_constraint(fkey['name'], type_="foreignkey")
+            else:
+                op.drop_constraint(fkey['name'], 'known_event', type_="foreignkey")
```

Comment:
Won't `batch_alter_table` work for all? It seems to work for mysql at least:
https://github.com/apache/airflow/blob/74da0faa7b85fe8a87d0a245179f8632fc93fd51/airflow/migrations/versions/74effc47d867_change_datetime_to_datetime2_6_on_mssql_.py#L52

Reply:
The main reason was 
`for fkey in inspector.get_foreign_keys(table_name="known_event", referred_table="users"):` was returning 2 elements, one of which had `fkey['name']` as `None` . And if you call `bop.drop_constraint(fkey['name'], type_="foreignkey")` for sqlite with `fkey['name']` it errors and complains about name not specified while it works for MySQL and Postgres -- so thought would be easier and safe to just keep the current behaviour with Postgres and MySQL.  (didn't investigate further on the foreign key entry with None name)

Reply:
Just tested it on Postgres too -- worked fine -- pushed the changes in https://github.com/apache/airflow/pull/13921/commits/890aa9336fa8d1ecf5f7305095bad44d9b961d97